### PR TITLE
RELATED: RAIL-3384 initialize dashboard plugin packages

### PR DESCRIPTION
**Pull request by no23reason**
Initializes the packages so that we have them in place and can start development.

---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3747/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/5369/ : SUCCESS in 12m 34s

File: tools/dashboard-plugin-template/package.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
+{
+    "name": "@gooddata/dashboard-plugin-template",
+    "version": "1.0.0",
+    "description": "GoodData SDK TypeScript & React skeleton",
```
Comment (lupko) [inline tools/dashboard-plugin-template/package.json:4]: pls fix package description

File: tools/dashboard-plugin-template/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "_note": "This file is only for IDE. For build use tsconfig.[dev/build].json (dev mode is less strict)",
+    "compilerOptions": {
+        "declaration": true,
```
Comment (lupko) [inline tools/dashboard-plugin-template/tsconfig.json:4]: missing `strict`. methinks the skels don't have it and probably should..

Comment (no23reason) [reply tools/dashboard-plugin-template/tsconfig.json:4]: Good catch, updated in both the pacakages and skeletons

File: tools/dashboard-plugin-template/package.json (hunk 1 above)
Comment (no23reason) [reply tools/dashboard-plugin-template/package.json]: Updated

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3755/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/5376/ : SUCCESS in 12m 44s

**Review by lupko — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/1097/ : SUCCESS in 4m 16s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1367/ : SUCCESS in 22s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/94/ : SUCCESS in 9m 08s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/5377/ : SUCCESS in 13m 16s

**Pull request merged**